src/ontology/mondo-edit.obo

```diff
@@ -503207,8 +503198,6 @@ xref: Orphanet:611216 {source="MONDO:equivalentTo"}
 xref: UMLS:C5436906 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1754257"}
 is_a: MONDO:0000159 {source="OMIM:619151"} ! bone marrow failure syndrome
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/253 {source="OMIM:619151"} ! ADH5
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/404 {source="OMIM:619151"} ! ALDH2
```

Comment:
This term has the annotation
`excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql`

These gene annotations should not be removed.